pants.ini

```diff
@@ -123,22 +121,52 @@ deps: ["3rdparty:thrift-0.9.2"]
 [compile.checkstyle]
 configuration: %(pants_supportdir)s/checkstyle/coding_style.xml
 
-[compile.scalafmt]
-skip: True
-
-[fmt.scalafmt]
-skip: True
+[compile.errorprone]
+excluded_targets: [
+                  'contrib/errorprone/tests/java/org/pantsbuild/contrib/errorprone:error',
+                  'testprojects/src/java/org/pantsbuild/testproject/printversion:printversion',
+                  'testprojects/src/java/org/pantsbuild/testproject/annotation/main:main',
+                  'testprojects/src/java/org/pantsbuild/testproject/annotation/processor:processor',
+                  ]
```

Comment:
This indentation is odd... I'm fine with the re-ordering, but could you undo the indentation changes in here?